Sources/RunnerLib/GetDangerJSPath.swift

```diff
@@ -10,6 +10,19 @@ public func getDangerCommandPath(logger: Logger,
         return args[dangerJSPathOptionIndex + 1]
     } else {
         logger.debug("Finding out where the danger executable is")
-        return try shellOutExecutor.spawn("command -v danger", arguments: []).trimmingCharacters(in: .whitespaces)
+
+        if let dangerJsPath = try? shellOutExecutor.spawn("command -v danger-js", arguments: []).trimmingCharacters(in: .whitespaces),
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/RunnerLib/GetDangerJSPath.swift;
  Line: 14;
-->

- :warning: Line should be 120 characters or less: currently 134 characters (`line_length`)